api/app/pagination.py

```diff
@@ -1,7 +1,44 @@
+from collections import OrderedDict
+
 from rest_framework.pagination import PageNumberPagination
+from rest_framework.response import Response
+from rest_framework.utils.urls import replace_query_param
 
 
 class CustomPagination(PageNumberPagination):
     page_size = 999
     page_size_query_param = "page_size"
     max_page_size = 999
+
+
+class IdentityPagination(CustomPagination):
+    def get_paginated_response_dynamo(
```

Comment:
What benefits do we get from the pagination if we're just defining a custom method that we call manually? Is it just the page size parameter? Can we actually write a proper pagination class and defer a lot of the logic to this class instead of the view?